features/src/autogluon/features/generators/datetime.py

```diff
@@ -39,42 +39,54 @@ def _transform(self, X: DataFrame) -> DataFrame:
 
     @staticmethod
     def get_default_infer_features_in_args() -> dict:
-        return dict(required_raw_special_pairs=[
-            (R_DATETIME, None),
-            (None, [S_DATETIME_AS_OBJECT])
-        ])
+        return dict()
 
     def _compute_fillna_map(self, X: DataFrame):
         fillna_map = dict()
         for datetime_feature in self.features_in:
-            datetime_series = pd.to_datetime(X[datetime_feature], errors='coerce')
+            if type(X[datetime_feature].iloc[0]) == pd.Timestamp:
```

Comment:
Why adding if statements? What if the type is not timestamp?

Reply:
Deleted if statement